test/generated-pkg/app-lib.js

```diff
@@ -45,6 +45,11 @@ describe('library package', function () {
       done()
     })
 
+    it('xqs does not call app module', function (done) {
+      assert.noFileContent('test/xqs/test-suite.xql', 'import module namespace app')
```

Comment:
Maybe I am not understanding this correctly, but it seems as if no test-suite.xql is added for libraries?

Reply:
Not quite, the file `test-suite.xql` does not include an import statement for `App` there is a corresponding positive check if the file is there, and in other packages that it should include something